packages/playground/webpack.config.dist.js

```diff
@@ -53,24 +53,14 @@ module.exports = {
         ],
       },
       {
-        test: /\.less$/,
-        include: /node_modules[\\/]antd/,
-        use: [
-          {
-            loader: "style-loader",
-            options: {
-              insert: "#antd-styles-iframe"
-            }
-          },
-          "css-loader",
-          {
-            loader: 'less-loader',
-            options: {
-              javascriptEnabled: true,
-            },
-          },
```

Comment:
I guess you removed less?

Reply:
Yeah, it appears to still work without it, so I guess we're OK to just not have it anymore??

Reply:
Does it cause problems if you leave it in?